plugins/osc/README.developer

```diff
@@ -1,3 +1,6 @@
+OSC Plugin Developer Information
```

Comment:
Is the file Markdown, or just something that looks similar? If the former, can we give it a .md extension.